app/services/daily_usages/fill_history_service.rb

```diff
@@ -18,6 +18,7 @@ def call
         datetime = date.in_time_zone(subscription.customer.applicable_timezone).beginning_of_day.utc
 
         next if date == Time.zone.today ||
+          subscription.daily_usages.where(usage_date: date - 1.day).exists? ||
```

Comment:
I'm wondering if the `DailyUsage.refreshed_at_in_timezone(datetime).where(subscription_id: subscription.id).exists?` is still relevant here

Reply:
yep - would like to remove it (and the other references) but on a next PR - when `usage_date` is populated